testing/src/test/java/org/motechproject/nms/testing/it/region/LanguageLocationCodesImportServiceBundleIT.java

```diff
@@ -255,4 +265,121 @@ private void assertLanguageCode(Language language, String code, String languageN
         assertNotNull(language.getName());
         assertEquals(languageName, language.getName());
     }
+
+    /**
+     * Method used to import CSV File For Location Data i.e circle, state and
+     * district & language location code
+     */
+    private HttpResponse importCsvFileForLocationData(String location,
+            String fileName)
+            throws InterruptedException, IOException {
+        HttpPost httpPost;
+        if (null==location) {
+            httpPost = new HttpPost(String.format(
+                    "http://localhost:%d/region/languageLocationCode/import",
+                    TestContext.getJettyPort()));
+        } else {
+            httpPost = new HttpPost(String.format(
+                    "http://localhost:%d/region/data/import/%s",
+                    TestContext.getJettyPort(), location));
+        }
+        FileBody fileBody = new FileBody(new File(String.format(
+                "src/test/resources/csv/%s", fileName)));
+        MultipartEntityBuilder builder = MultipartEntityBuilder.create();
+        builder.setMode(HttpMultipartMode.BROWSER_COMPATIBLE);
+        builder.addPart("csvFile", fileBody);
+        httpPost.setEntity(builder.build());
+        HttpResponse response = SimpleHttpClient.httpRequestAndResponse(
+                httpPost, RequestBuilder.ADMIN_USERNAME,
+                RequestBuilder.ADMIN_PASSWORD);
+        return response;
+    }
+
+    /**
+     * 518: To verify LLC is set successfully for all the districts in a state.
+     * <p>
+     * 519: To verify that multiple states can be mapped to one circle.
+     * <p>
+     * 520: To verify that one state can be mapped to more than one circle.
+     */
+    @Test
+    public void verifyFT518_FT519_FT520() throws InterruptedException,
```

Comment:
This should be split up into multiple tests since the imports are logically separate operations for the datasets. Any reason why they are combined into one?